Source/Resources/Content/PowerShell/Exchange/Organization/OrganizationConfig/First Organization.json

```diff
@@ -25,6 +25,7 @@
   "defaultPublicFolderMaxItemSize": "Unlimited",
   "defaultPublicFolderMovedItemRetention": "7.00:00:00",
   "defaultPublicFolderProhibitPostQuota": "2 GB (2,147,483,648 bytes)",
+  "description":"What is it: Defines various Exchange settings. \n Simeon Notes: Microsoft changes these setting frequently as features are added and removed. \n Learn More: https://docs.microsoft.com/en-us/powershell/module/exchange/set-organizationconfig?view=exchange-ps",
```

Comment:
**What does this do?**
Defines various Exchange settings. Microsoft changes these settings frequently as features are added and removed. @metamenos I feel like we need to say something more here although we aren't changing the default

**Why should you use this?**
If you want to track configuration changes made in the environment.

**What is the end-user impact?**
N/A

**Learn more**
[Set-OrganizationConfig](https://docs.microsoft.com/en-us/powershell/module/exchange/set-organizationconfig?view=exchange-ps)